wgpu/Cargo.toml

```diff
@@ -65,13 +64,13 @@ webgl = ["dep:hal", "wgc/gles"]
 # --------------------------------------------------------------------
 
 ## Enable accepting SPIR-V shaders as input.
-spirv = ["naga/spv-in", "wgc/spirv"]
+spirv = ["naga/spv-in", "wgpu-core?/spirv"]
 
 ## Enable accepting GLSL shaders as input.
-glsl = ["naga/glsl-in", "wgc/glsl"]
+glsl = ["naga/glsl-in", "wgpu-core?/glsl"]
```

Comment:
subtle: we dropped a `?` here, and previously would always get wgpu-core even if webgl wasn't enabled.